### ISSUE TITLE
Parse Tweet Coordinate

### ISSUE THREAD

**Issue by enj** [ChimeraCoder/anaconda#56]
Hello,

I am relatively new to Go so please forgive me if this is a noob question.

I am trying to parse the coordinates out of a tweet.  Currently I have the following code:
```go
for _, tweet := range tweets {
	p := tweet.Coordinates.(map[string]interface {})
	c := p["coordinates"].([]interface{})
	lat := c[1].(float64)
	lng := c[0].(float64)
```
While this works fine I am wondering if there is a more idiomatic / type safe way too do this - perhaps with a struct like this (though I have no idea how to use this struct):
```go
type geoJSON struct {
	Coordinates []float64 `json:"coordinates"`
	Type        string    `json:"type"`
}
```

**Comment by eseglem** [comment]
I have been dealing with Twitter for quite some time now in python, and I am experimenting with this package to move it all to go. I primarily care about the geotagged data so this was the first change I made when I forked it.

Twitter will never pass anything other than a two digit coordinate with the type of point in the current version of the API. As per here: https://dev.twitter.com/overview/api/tweets#obj-coordinates

I have been running this struct for a while now without any errors for coordinates. 

```go
	Coordinates  struct {
		Coordinates [2]float64 `json:"coordinates"`
		Type        string     `json:"type"`
	} `json:"coordinates"`
```

I have attempted to add a few help functions as well:
HasCoordinates() which returns if the tweet object has coordinates. 
Latitude() and Longitude() Which return the coordinate rather than remembering the order they are in the list. The problem I am having with this one, is return value. I think I need to raise an error if coordinates do not exist.
